File: Upcoder/Lập trình cơ bản/5.Mảng/Thêm phần tử/MANGCS1.py
def main():
    # Nhập x và k
    x, k = map(int, input().split())
    
    # Nhập mảng a
    a = list(map(int, input().split()))
    n = len(a)
    
    # Xuất mảng với k được chèn vào vị trí x
    for i in range(n + 1):  # Tăng n lên 1 để xử lý trường hợp x == n
        if i == x:
            print(k, end=" ")
        if i < n:
            print(a[i], end=" ")
if __name__ == "__main__":
    main()

File: Upcoder/Lập trình cơ bản/5.Mảng/Thêm phần tử/MANGCS1.cs
using System;
namespace MANGCS1{
    public class Program
    {
        static void Main()
        {
            string[] input = Console.ReadLine().Split(' ');
            int x = int.Parse(input[0]);
            int k = int.Parse(input[1]);
    
            string[] arrInput = Console.ReadLine().Split(' ');
            int n = arrInput.Length;
            int[] a = new int[n];
    
            for (int i = 0; i < n; i++)
            {
                a[i] = int.Parse(arrInput[i]);
            }
    
            for (int i = 0; i <= n; i++) 
            {
                if (i == x)
                {
                    Console.Write(k + " ");
                }
                if (i < n)
                {
                    Console.Write(a[i] + " ");
                }
            }
        }
    }
}